rabbitmq-ballerina/Ballerina.toml

```diff
@@ -24,10 +24,10 @@ name = "system"
 version = "@stdlib.system.version@"
 
 [[platform.java11.dependency]]
-path = "./lib/amqp-client-5.7.3.jar"
+path = "./lib/amqp-client-5.10.0.jar"
```

Comment:
I highly recommend to define the version numbers in the `gradle.properties` and use it here, instead of hard-coding the version directly in the toml file. This is pretty much similar to the stdlib versions or project version we use in the toml file.

Reply:
Fixed